srcpkgs/ampache/template

```diff
@@ -1,16 +1,17 @@
 # Template file for 'ampache'
 pkgname=ampache
-version=5.5.1
+version=5.6.1
 revision=1
 create_wrksrc=yes
 hostmakedepends="unzip"
-depends="php8.1 mariadb"
+depends="php8.2 mariadb"
```

Comment:
The PHP version bump is not required. Do we want the latest supported version?

Reply:
Makes sense in my opinion, later versions tend to be more secure so the bump should only have positive impacts (if there are any notable differences at all).